src/sagemaker/lineage/query.py

```diff
@@ -259,23 +285,182 @@ def _get_app(self, elements):
                                 "label": "data(label)",
                                 "color": "gray",
                                 "text-halign": "left",
-                                "text-margin-y": "3px",
-                                "text-margin-x": "-2px",
-                                "font-size": "3%",
-                                "width": "1%",
-                                "curve-style": "taxi",
+                                "text-margin-y": "2.5",
+                                "font-size": "3",
+                                "width": "1",
+                                "curve-style": "bezier",
+                                "control-point-step-size": "15",
                                 "target-arrow-color": "gray",
                                 "target-arrow-shape": "triangle",
                                 "line-color": "gray",
                                 "arrow-scale": "0.5",
+                                "font-family": "verdana",
                             },
                         },
+                        {"selector": ".Artifact", "style": {"background-color": "#146eb4"}},
```

Comment:
`vert.lineage_entity` below is what is used to name these selectors? Can we use the same lineage_entity type directly here? 

Right now we use these colors both here and in the self.html.Divs below. If we need it in two places can we use a variable? However, would it be possible to apply the style using the selector name instead?

Reply:
I don't think `DashVisualizer` class has the visibility of `vert.lineage_entity`, and I created `self.entity_color` dictionary so that changing colors or adding entities would be easier and more code structure would be more modularized